velox/core/QueryConfig.h

```diff
@@ -339,6 +339,10 @@ class QueryConfig {
   static constexpr const char* kSparkLegacyStatisticalAggregate =
       "spark.legacy_statistical_aggregate";
 
+  /// If true, ignore null fields when generating JSON string.
+  static constexpr const char* kSparkJsonIgnoreNullFields =
+      "spark.json_ignore_null_fields";
```

Comment:
Would you please document this config in https://github.com/facebookincubator/velox/blob/main/velox/docs/configs.rst?